app/domain/authentication/authn_jwt/identity_providers/fetch_identity_path.rb

```diff
@@ -0,0 +1,82 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+    module IdentityProviders
+      # Fetch the identity path from the JWT authenticator policy of the host identity
+      FetchIdentityPath = CommandClass.new(
+        dependencies: {
+          resource_class: ::Resource,
+          fetch_required_secrets: ::Conjur::FetchRequiredSecrets.new,
+          logger: Rails.logger
+        },
+        inputs: %i[authentication_parameters]
+      ) do
+        def call
+          fetch_identity_path
+        end
+
+        private
+
+        def fetch_identity_path
+          @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingIdentityPath.new)
+          validate_configuration_and_set_identity_path_value
```

Comment:
Move validate and logs to call functioion

Reply:
as we agreed i ignore this